lib/node_modules/@stdlib/blas/ext/base/dnannsumpw/lib/dnannsumpw.js

```diff
@@ -55,20 +56,10 @@ var sumpw = require( './sumpw.js' );
 function dnannsumpw( N, x, strideX, out, strideOut ) {
 	var ix;
 	var io;
-	if ( strideX < 0 ) {
-		ix = (1-N) * strideX;
-	} else {
-		ix = 0;
-	}
-	if ( strideOut < 0 ) {
-		io = -strideOut;
-	} else {
-		io = 0;
-	}
-	out[ io ] = 0.0;
-	out[ io+strideOut ] = 0;
```

Comment:
@headlessNode This logic was intentional. Namely, the output array needs to be initialized. Otherwise, someone could provide an array of random values and get back erroneous results.